clever_test.go

```diff
@@ -133,6 +133,53 @@ func TestQueryTeachers(t *testing.T) {
 	}
 }
 
+func TestQueryEvents(t *testing.T) {
+	clever := New(mock.NewMock(nil, "./data"))
+	results := clever.QueryAll("/v1.1/events", nil)
+	if !results.Next() {
+		t.Fatal("Found no events")
+	}
+	event := Event{}
+	if err := results.Scan(&event); err != nil {
+		t.Fatalf("Error retrieving event: %s\n", err)
+	}
+
+	resp := EventResp{}
+	if err := clever.Query(fmt.Sprintf("/v1.1/events/%s", event.Id), nil, &resp); err != nil {
+		t.Fatalf("Error retrieving event: %s\n", err)
+	}
+	expectedEvent := Event{
+		Type:    "teachers.deleted",
+		Created: "2015-07-27T19:38:24.919Z",
+		Id:      "55b688b1cd921d4a081c4ec3",
+		Data: map[string]interface{}{
+			"object": map[string]interface{}{
```

Comment:
Re conversation about this, while it is maybe more convenient to factor this out `object` step out, if we want this library to closely follow the api, maybe that would just be confusing. Leaving in for the moment.